### ISSUE TITLE
Featured Brandfie Is Not Good

### ISSUE THREAD

**Issue by ahmadawais** [ahmadawais/brandfie#5]
@dhanishgajjar We need a better featured #brandfie than this one

![](https://i.imgur.com/mSoYVTy.png)

**Comment by dhanishgajjar** [comment]
Yeah, I am working on that now .. that and the logo ... feel free to update the text everywhere ...  once that is done .. and only stripe will remain

**Comment by ahmadawais** [comment]
I'll let you work on it for now. Have a lot on my plate, a meetup I am organizing this weekend and other bank related stuff. I'll jump in a few days and by then you'll have a much polished site. Feel free to ask anything!
